### ISSUE TITLE
Ändra befintliga tabeller

### ISSUE THREAD

**Issue by MalinAhlberg** [spraakbanken/karp-mfl#6]
Gör det tydligt att man ändrar en befintlig ingång och inte skapar en ny.
Hur kommer man till den här vyn?

**Comment by kristiank** [comment]
Från ordlistan? (alltså inte kandidatlistan)

**Comment by MalinAhlberg** [comment]
Ja, precis. Från kandidatlistan ändrar man väl alltid befintliga?

**Comment by kristiank** [comment]
Så vitt jag förstått så är kandidatlistan en lista av ord man vill lägga till i Karp tillsammans med rätt morfologisk information. Kandidatlistan är då en lista med **o-befintfliga** ord, men som man det har automatiskt lagts till en eller flera gissningar på matchande **befintliga** paradigm.

Enligt min uppfattning finns det då två entiteter som kan kallas _befintlig tabell_:
* **paradigm** alltså abstrakta böjningsmönster (á la `1 + t + 2`)
* **ord/leksem** alltså specifika ordboksartiklar med tillhörande abstrakt böjningsmönster (á la `tüt + t + ö`)

Från kandidatlistan kan man ju då antingen:
1. klicka på en gissning (alltså ett befintligt paradigm) och se över den specifika böjningstabellen och vara nöjd med den och bara klicka på knappen för "lägg till som ordboksartikkel i Karp"
2. klicka på en gissning och inse att det är fel i böjningstabellen och ändra i tabellen

Vad man under punkt 2. då gör är att förse pextract med ny input som kan leda till ett ekstraherandet av ett nytt paradigm. Dock kan ändringen i böjningstabellen leda till att den matchar ett annat befintligt, alltså av pextract redan funnet paradigm, då extraheras inte ett nytt paradigm utan istället paras ordet med det andra paradigmet istället för det första, och läggs till i Karp.

Jag är förvirrad, menar du med _befintlig tabell_:
1. en specifik ordboksartikkels böjningstabell, alltså en specifik variabelinstansiering av ett befintligt paradigm, eller
2. att man vill ändra ett befintligt abstrakt böjningsmönster? Och att ändringarna då skulle råda för alla leksem som redan angivits med det abstrakta böjningsmönstret?
